#servo motor values in angles (0 - 180)
int16 servoFLAngle
int16 servoFRAngle
int16 servoRLAngle
int16 servoRRAngle